packages/editable-html/src/plugins/list/__test__/index.test.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+
+import {configure} from "enzyme";
+import Adapter from "enzyme-adapter-react-15";
+import renderer from 'react-test-renderer';
+import List from './../../list/index';
+import Util from '../../utils';
+
+configure({adapter : new Adapter() });
+
+describe("ListPlugin", () => {
+  let list, ListPlugin, toolbar;
+  beforeEach(() => {
+    list = require('../index');
```

Comment:
this is unnecessary - you've already imported `List` and co above.

Reply:
Removed